File: 프로그래머스/lv0/120839. 가위 바위 보/가위 바위 보.js
function solution(rsp) {
    var answer = rsp.split('')
    var result = ''
    
    for(let i of answer) {
        if(i == 2) {
            result += 0
        } else if (i == 0) {
            result += 5
        } else {
            result += 2
        }
    }
    return result;
}

File: 프로그래머스/lv0/120839. 가위 바위 보/가위 바위 보.java
class Solution {
    public String solution(String rsp) {
        String[] answer = rsp.split("");
        String result = "";

        for (String s : answer) {
            switch (s) {
                case "5":
                    s = "2";
                    break;
                case "2" :
                    s = "0";
                    break;
                case "0" :
                    s = "5";
                    break;
            }
            result += s;
        }
        return result;
    }
}